src/box/request.h

```diff
@@ -54,7 +55,8 @@ struct tuple;
  */
 int
 request_create_from_tuple(struct request *request, struct space *space,
-			  struct tuple *old_tuple, struct tuple *new_tuple);
+			  struct tuple *old_tuple, struct tuple *new_tuple,
+			  struct region *region);
```

Comment:
Nit: You forgot to update the comment to this function.

Reply:
Fixed.